contracts/RiskManagerV1.sol

```diff
@@ -38,9 +38,9 @@ interface IDeposit {
 
```

Comment:
Not in this PR but just noticed it - we could do something to remove `notifyRedemptionSignatureTimedOut` from `IDeposit` interface. This function is called only in tests and ideally, `IDeposit` should have only those functions the risk manager is interested in.

Reply:
In one of the previous PRs I introduced `IDepositStub` interface exactly for that purpose. However, we decided to move all real `Deposit` methods to the `IDeposit` interface, even if they are not needed by contracts.

Reply:
Hmm... I think we could have a misunderstanding. If I am not mistaken we were using `DepositStub` and we were additionally adding `IDepositStub`. Now we have a system test where we cannot use `DepositStub` (or at least, we shouldn't be).

Anyway, it does not matter now. 

If we introduce this interface in `contracts/test/DepositStub.sol`, document that those are the functions from tBTC Deposit but used only for tests, and even maybe consider naming this interface just `IDeposit` (?) this will be better than what we have now IMO. What do you think?

Anyway, no reason to block this PR, let's see what we can do about it separately.

Reply:
Yep, we could try it as you proposed.